web/handlers_test.go

```diff
@@ -436,7 +436,7 @@ func TestGenerateDevCSP(t *testing.T) {
 
 		devCSP := generateDevCSP(*c)
 
-		assert.Equal(t, " 'unsafe-eval' 'unsafe-inline'", devCSP)
+		assert.Equal(t, "'unsafe-inline' 'unsafe-eval'", devCSP)
```

Comment:
This order now matches the order that they flags are passed in `DeveloperFlags`. The preceding space is also added outside of this method